caluma/caluma_form/tests/test_option.py

```diff
@@ -50,26 +50,3 @@ def test_copy_option(db, option, schema_executor):
     assert new_option.label == "Test Option"
     assert new_option.meta == option.meta
     assert new_option.source == option
-
-
-def test_dynamic_option(db, schema_executor, dynamic_option_factory):
-    query = """
-        query dynamicOptions ($filter:[DynamicOptionFilterSetType]) {
-          allUsedDynamicOptions(filter: $filter){
-            edges{
-              node{
-                slug
-                label
-              }
-            }
-          }
-        }
-    """
-
-    dynamic_option = dynamic_option_factory()
-    inp = {"filter": [{"document": dynamic_option.id}]}
-    result = schema_executor(query, variable_values=inp)
-
-    assert not result.errors
-    assert dynamic_option.slug == "service-bank-arm"
-    assert dynamic_option.label["en"] == "Calvin Graham"
```

Comment:
Why was this test deleted?

Reply:
The test would really only test if the query allUsedDynamicOptions works, which is verified in other places already, the involved filters are trivial.

Now the current  filter also doesn't make sense and the test doesn't do anything with the returned values, hence it's useless.

Reply:
Alright :+1: